v3/cmd/zlint/main.go

```diff
@@ -47,6 +47,8 @@ var ( // flags
 	includeSources  string
 	excludeSources  string
 	printVersion    bool
+	config          string
```

Comment:
The changes in `main` are to:

1. Add the new flags that either point to a configuration file are prints an example default one to the terminal.
2. If printing the default example is selected, then that is done and the problem exists. Any other flag is ignored. This behavior is consistent with printing a list of all available lints as well as printing ZLint's version.